pipeline/src/test_util.rs

```diff
@@ -516,3 +518,19 @@ pub fn run_reparse_test_with_blacklist(file: &str, blacklist: &[&str]) {
         .compute_optimized_pil()
         .unwrap();
 }
+
+// TODO: Add #[cfg(feature = "stwo")] to conditionally compile this function. now when I add it, this code is disabled. how to eable by default?
```

Comment:
Yes we should have this. Check the other cases above for example for plonky3, it has a version with and without the feature, where the one without the feature is a no-op.
What do you mean by the code being disabled? If it shows as disabled in your IDE it just means that the feature flag is not active, which would make sense if stwo is off by default.